peer-to-peer-scheduler/peer-to-peer-scheduler.gs

```diff
@@ -0,0 +1,900 @@
+const MEETING_LINK_INDEX = 12;
+const CANCELLATION_DATES_INDEX = 13;
+const RECORDING_LINKS_INDEX = 14;
+const CHAT_LINKS_INDEX = 15;
+/**
+ * If you add/remove/modify columns, you must also change
+ * 1. DATA_END_COLUMN
+ * 2. MSG_HELP
+ */
+
+const DATA_BEGIN_ROW = 2;
+const DATA_END_COLUMN = "P";
+
+const FLAG_IGNORE_ROW = 0;
+const FLAG_SEND_INVITE = 1;
+const FLAG_CANCEL_INVITE = 2;
+
+const NOT_FOUND = "NOT FOUND";
+const RECORDING_MIME_TYPE = "video/mp4";
+const CHAT_MIME_TYPE = "text/plain";
+
+const MENU_TITLE = "10x P2P";
+const MENU_INVITES_TO_STUDENTS = "Send invites to students";
+const MENU_INVITES_TO_MENTORS = "Send invites to mentors";
+const MENU_GET_RECORDING_AND_CHAT_LINKS = "Get recording and chat links"
+const MENU_CANCEL_MEETINGS = "Cancel meetings";
+const MENU_HELP = "Help";
+
+const TITLE_ERROR = "Error";
+const TITLE_SUCCESS = "Success";
+const TITLE_HELP = "Help";
+
+const MSG_CALENDAR_REJECTED = "You chose not to use the current calendar.";
+const MSG_CALENDAR_NOT_FOUND = "Calendar not found";
+const MSG_COULD_NOT_FETCH_DATA = "Unable to fetch data from the sheet";
+const MSG_HELP = `
+Version: ${LAST_PUBLISHED_VERSION + 1}
+- Your sheet should be in the following format
+- 1st row must contain column names
+- Actual data must start from 2nd row
+
+- Your sheet must have the following columns in the given order
+
+1. Meeting Type - To be filled by the team.
+2. Group Number - To be filled by the team.
+3. Module Name - To be filled by the team.
+4. Student Emails - Email ids of the students - Comma separated values. To be filled by the team.
+5. SSM Emails - Additional Emails you want to include in the meeting - Comma separated values.
+6. Mentor Email - Email id of the mentor. To be filled by the team.
+7. Ops Request - (0 - ignore the row),(1 - for meet invites),(2 - for Cancel Meeting) To be filled by the team.
+8. Start Date - Start Date of Meeting in yyyy-mm-dd format. To be filled by the team.
+9. End Date - End Date of Meeting in yyyy-mm-dd format. To be filled by the team.
+10. Start time - Start time of Meeting in hh:mm format (24 hours). To be filled by the team.
+11. End time - End time of Meeting in hh:mm format (24 hours). To be filled by the team.
+12. Meeting ID - Auto populated by the script.
+13. Meeting link - Auto populated by the script.
+14. Cancellation Dates - Dates in yyyy-mm-dd format - Comma separated values. To be filled by the team.
+15. Recording Links - Auto populated by the script.
+16. Chat Links - Auto populated by the script.
+
+NOTE:
+If you want to cancel full invite for a row:
+- Give Ops Request as 2 for that row and 
+- keep Cancellation Dates (12th Column) cell empty for that row
+
+Meeting Title Format:
+- Meeting Type | Group Number | Module Name - 10x Academy
+`;
+
+const DATE_DELIMITER = "-";
+
+const MEETING_TITLE_DELIMITER = " | ";
+const MEETING_TITLE_SUFFIX = " - 10x Academy"
+const MEETING_FIRST_REMINDER_MINUTES = 60;
+const MEETING_SECOND_REMINDER_MINUTES = 30;
```

Comment:
`MEETING_FIRST_REMINDER_MINUTES` and `MEETING_SECOND_REMINDER_MINUTES` are not used anywhere.

Reply:
Removed `MEETING_FIRST_REMINDER_MINUTES` & `MEETING_SECOND_REMINDER_MINUTES`

As we can make `defaultRemainders` for all meets sent through a calendar in their calendar settings.

For CodeAlong we are using the following remainders:
`defaultReminders: 
   [ { minutes: 5, method: 'popup' },
     { minutes: 10, method: 'popup' },
     { minutes: 15, method: 'popup' },
     { minutes: 30, method: 'popup' } ]`
     
    
![image](https://user-images.githubusercontent.com/98754869/231598155-708ae6e6-539a-4aad-a64c-a5f9c3143f75.png)